src/main/java/frc/robot/subsystems/ramp/RampIOTalonFX.java

```diff
@@ -29,15 +43,25 @@ public void setPID(double kP, double kI, double kD) {
     controller.setD(kD);
   }
 
+  @Override
+  public void addOffset(double offset) {
+    angle_offset += offset;
+  }
```

Comment:
No change for now since you have something that works.

we probably should have made this take no argument and simply set angle_offset to `talon.getPosition().getValueAsDouble()` in radians

 Just noting that this is a more readable method for future homing code